bokeh/layouts.py

```diff
@@ -204,12 +207,21 @@ def layout(children=None, sizing_mode='fixed', responsive=None, *args):
                     """Only LayoutDOM items can be inserted into a layout.
                     Tried to insert: %s of type %s""" % (item, type(item))
                 )
-        rows.append(row(children=row, sizing_mode=sizing_mode))
+        rows.append(row(children=row_children, sizing_mode=sizing_mode))
     grid = column(children=rows, sizing_mode=sizing_mode)
     return grid
 
 
-def gridplot(children=None, toolbar_location='left', sizing_mode='fixed', responsive=None, toolbar_options=None, *args):
+def chunks(l, n):
+    """Yield successive n-sized chunks from list, l."""
+    for i in range(0, len(l), n):
+        yield l[i: i+n]
```

Comment:
I didn't see it: https://docs.python.org/2/library/itertools.html

but may be missing something.